src/event_emitter.cr

```diff
@@ -1,4 +1,26 @@
 require "./event_emitter/*"
 
 module EventEmitter
+    alias Any = Void |
+        Nil |
+        Bool |
+        Int16 |
+        Int32 |
+        Int64 |
+        Float32 |
+        Float64 |
+        String |
+        Array(Any) |
+        Hash(Any, Any)
+
+    # TODO: See if there isn't a better way to do this
+    def self.any(any)
```

Comment:
I'm a little worried about the performance issue that may be raised by using those typecastings, but IMO it's justifiable by the flexibility it brings, so it's a reasonable tradeof.

I rewrote the if/else to case/when statements, WDYT?

```crystal
  def self.any(any)
    case any
    when Array
      any.map &.as(Any)
    when Hash
      any.map { |k, v| {k.as(Any), v.as(Any)} }.to_h
    else
      any
    end
  end
```

Reply:
I considered that as well, and imo it does look a bit cleaner. No problem with it from my end. I don't like the performance hit either, but for now I don't see any way around it unfortunately.